controllers/frontend_controller.go

```diff
@@ -136,6 +136,10 @@ func (r *FrontendReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, err
 	}
 
+	if frontend.Spec.Disabled {
```

Comment:
Should this have a default `false` value somewhere? Or will this resolve to false if empty? We don't have omitempty so I'm not sure what the deal would be. I think this is fine but wanted to ask.